dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+   specification, includes the PayID Easy Checkout URL representing a
+   resource on the wallet which can be used to complete a payment.
+
+   Payment receivers who wish to initiate an Easy Checkout flow MUST
+   query the sender's PayID Discovery server to obtain a PayID Easy
+   Checkout URL.  PayID servers that wish to enable PayID Easy Checkout
+   MUST include a JRD Link conforming to the definition in the JRD
+   section (Section 4) of this paper in all PayID Easy Checkout
+   Discovery responses.
+
+   Payment receivers SHOULD implement fallback measures to complete a
+   checkout flow if a user's wallet does not support PayID Easy
+   Checkout.
+
+   The following steps describe how a PayID client can query a PayID
+   server to obtain a PayID Easy Checkout URL.
+
+3.1.1.  Step 1: Assemble PayID Easy Checkout Discovery URL
+
+   The process of assembling a PayID Discovery URL is defined in section
+   4.1.1 of [PAYID-DISCOVERY], and is the same as for this protocol.
+
+3.1.2.  Step 2: Query PayID Easy Checkout Discovery URL
+
+   Querying the PayID Discovery URL is defined in section 4.1.2 of
+   [PAYID-DISCOVERY], and is performed in the same way as this protocol.
+
+   Clients SHOULD implement fallback measures to complete checkout if
+   the PayID Easy Checkout Discovery query fails.
+
+3.1.3.  Step 3: Parse PayID Easy Checkout Metadata
+
+   If PayID Easy Checkout is supported, a PayID Discovery server MUST
+   respond with an HTTP status code "200" and a JSON payload containing
+   a JSON Resource Descriptor (JRD) that contains a link conforming to
+   the JRD section (Section 4) of this document.
+
+   For example, a PayID server might respond to a PayID Discovery query
+   with the following payload:
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 4]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+    {
+       "subject": "payid:alice$wallet.com",
+       "links": [
+           {
+               "rel" : "https://payid.org/ns/payid-easy-checkout/1.0",
+               "href": "https://wallet.com/checkout"
+           }
+       ]
+    }
+
+   A PayID Discovery client MUST parse this response to find the PayID
+   Easy Checkout Link.  If the JRD returned from the PayID Discovery
+   query does not contain a PayID Easy Checkout Link in its 'links'
+   collection, PayID Easy Checkout is considered to have failed.  Once a
+   PayID Easy Checkout URL has been obtained from the PayID Easy
+   Checkout Link, PayID Easy Checkout Discovery is considered to be
+   complete.
+
+3.2.  PayID Easy Checkout URL Assembly
+
+   A PayID Easy Checkout URL represents the resource on a wallet that
+   can be used by a sender to complete a payment.  However, before
+   directing a payer to their sending client, the payment receiver MUST
+   append all of the query parameters defined in the following section
+   (Section 3.2.1).
```

Comment:
"wallet" alone does not tell if it is payer's wallet or recipient's wallet. So "wallet"--> "payer's wallet"
Who is the sending client?
Who is payment receiver? "Recipient"?